docs/integreatly/modules/ROOT/pages/ds-realtime.adoc

```diff
@@ -3,7 +3,7 @@ include::_partials/attributes.adoc[]
 // end::excludeDownstream[]
 
 [id='realtime_{context}']
-= Supporting real-time updates in your mobile app
+= Subscriptions - Supporting real-time updates in your mobile app
```

Comment:
I know that this is controversial but this is good example where docs obfuscate real knowledge. 
Subscriptions as single word means `Supporting real-time updates in your mobile app` so instead of long title we can have just subscriptions - same as in other chapters